exercises/atbash-cipher/atbash_cipher.py

```diff
@@ -1,6 +1,6 @@
-def encode(string):
+def encode(plain):
```

Comment:
For me `plain_text` and `ciphered_text` looks a bit better. 
But your suggestion is also Ok

Reply:
@m-a-ge Your suggestion is better. I've changed them to `plain_text` and `ciphered_text`